CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [1.8.0]
+### Changed
+* Improved error messages when Earthdata user must select Study Area or accept EULA.
+
```

Comment:
I'd like to see a note here about how we've updated our testing approach to use mocks.

Reply:
Hmm, I've usually avoided putting stuff in the Changelog that's related to refactoring or "internal" changes that users won't care about. There's some interesting discussion on this point [here](https://github.com/olivierlacan/keep-a-changelog/discussions/463).

However, in this case, it actually does change our public API because the `hyp3_sdk.TESTING` global variable no longer exists. Technically I think that means this should be a major release version, but I really hope no users were relying on that variable.

Anyway, I will go ahead and add it to the Changelog.